osbs/cli/main.py

```diff
@@ -80,14 +81,22 @@ def _get_build_metadata(pipeline_run, user_warnings_store):
 
 def print_output(pipeline_run, export_metadata_file=None):
     user_warnings_store = UserWarningsStore()
-    _print_pipeline_run_logs(pipeline_run, user_warnings_store)
+    get_logs_failed = _print_pipeline_run_logs(pipeline_run, user_warnings_store)
```

Comment:
Misleading, because getting the logs *technically* fails when `pipeline_run_logs` is *not* an instance of collections.abc.Iterable

Reply:
I am not covering that case when it isn't even iterable because that is some weird corner case, I am covering ONLY when during get_logs openshift gives ConnectionError(ReadTimeoutError)